test/modules/aws-opensearch-integration.ts

```diff
@@ -0,0 +1,208 @@
+import { OpenSearch } from '@aws-sdk/client-opensearch';
+
+import * as iasql from '../../src/services/iasql';
+import {
+  defaultRegion,
+  execComposeDown,
+  execComposeUp,
+  finish,
+  getPrefix,
+  itDocs,
+  runBegin,
+  runCommit,
+  runInstall,
+  runQuery,
+} from '../helpers';
+
+const fetch = require('node-fetch');
```

Comment:
This isn't necessary in Node 18. If Jest is giving you issues with it not being here, you can [`declare` the `fetch` function similarly to this](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/14796/files#diff-0876e65c08560a069314bd2c7285249db01a47cfb3e8d6ee8f0ce71b7d46549eR111) perhaps replacing the arguments with `any` for simplicity.